pyaedt/modeler/GeometryOperators.py

```diff
@@ -1448,3 +1462,354 @@ def orient_polygon(x, y, clockwise=True):
             x.reverse()
             y.reverse()
         return x, y
+
+    @staticmethod
+    @aedt_exception_handler
+    def v_angle_sign(va, vb, vn, righthanded=True):
+        """Evaluate the signed angle between two geometry vectors.
```

Comment:
Include an empty line to get this to render correctly.
```suggestion
        """Evaluate the signed angle between two geometry vectors.

```

Reply:
done